controllers/productDetailCtrl.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+const express = require("express");
+const app = express();
+const passport = require("passport");
+
+app.set("models", require("../models"));
```

Comment:
the app.set should only really happen in the app.js, so you can require it through the req after that

Reply:
Hey @Rian501, thanks for the great suggestion! Per your request, I have updated our CTRL to correctly use req.app to achieve the same effect. Let me know if you have any questions!